libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/internal/MapboxOnboardRouter.kt

```diff
@@ -34,42 +29,22 @@ import kotlinx.coroutines.withContext
  * It uses offline storage path to store and retrieve data, setup endpoint,
  * tiles' version, token. Config is provided via [OnboardRouterOptions].
  *
- * @param applicationContext Application context
- * @param accessToken Mapbox token
  * @param navigatorNative Native Navigator
- * @param options configuration for on-board router
  * @param logger interface for logging any events
- * @param skuTokenProvider skuTokenProvider [SkuTokenProvider]
  */
 @MapboxModule(MapboxModuleType.NavigationOnboardRouter)
 class MapboxOnboardRouter(
-    private val applicationContext: Context,
-    private val accessToken: String,
     private val navigatorNative: MapboxNativeNavigator,
-    private val options: OnboardRouterOptions,
-    private val logger: Logger,
-    private val skuTokenProvider: SkuTokenProvider
+    private val logger: Logger
 ) : Router {
 
     private companion object {
         private val loggerTag = Tag("MapboxOnboardRouter")
-        private const val USER_AGENT: String = "MapboxNavigationNative"
-        private const val THREADS_COUNT = 2
     }
 
     private val mainJobControl by lazy { ThreadController.getMainScopeAndRootJob() }
     private val gson = Gson()
 
-    private val onboardRouterFiles = OnboardRouterFiles(applicationContext, logger)
-
-    init {
-        mainJobControl.scope.launch {
-            onboardRouterFiles.absolutePath(options)?.let { absolutePath ->
-                configureRouter(absolutePath)
```

Comment:
Saw that this was removed from `MapboxOnboardRouter` but not from `MapboxOnboardRouterTest`. E.g. https://github.com/mapbox/mapbox-navigation-android/blob/6c03d40997e50051999591f4a160673db0f8ca93/libdirections-onboard/src/test/java/com/mapbox/navigation/route/onboard/internal/MapboxOnboardRouterTest.kt#L70 Could you follow up with a quick PR adapting the test when you have a chance @korshaknn?